src/pages/groupComparison/NumericalDataVisualisation.tsx

```diff
@@ -0,0 +1,66 @@
+import BoxScatterPlot, {IBoxScatterPlotProps, toBoxPlotData} from "shared/components/plots/BoxScatterPlot";
```

Comment:
i think NumericalDataVisualization is to general a name and suggests that this could/should be reused somewhere. really i think it's an organizational wrapper to avoid bloating Clinicaldata.tsx file.    so maybe ClinicalDataBoxPlotWrapper.tsx?

Reply:
Hmm, a table is not a boxplot. And indeed, `NumericalDataVisualization` is to general a name. Maybe a combination:
`ClinicalNumericalDataVisualization`? 
Not pretty but more expressive than `ClinicalDataBoxPlotWrapper`

Reply:
renamed in [b30c1f0](https://github.com/cBioPortal/cbioportal-frontend/pull/4248/commits/b30c1f0b3c997111fb48c27d3442e56a2eb5c04e)